src/shared/components/simple-select/SimpleSelect.tsx

```diff
@@ -40,6 +41,7 @@ type HTMLSelectProps = SelectHTMLAttributes<HTMLSelectElement>;
 export type Option = {
   value: string;
   label: string;
+  isSelected?: boolean;
```

Comment:
Why is this needed? This adds confusion to the component's api, by introducing competition between `option.isSelected` and `props.value` (and by providing to limit to how many options can have the `isSelected` flag on them).